src/test/scala/com/mozilla/telemetry/streaming/TestErrorAggregator.scala

```diff
@@ -386,7 +386,9 @@ class TestErrorAggregator extends FlatSpec with Matchers {
 
       override def onQueryTerminated(event: StreamingQueryListener.QueryTerminatedEvent): Unit = {
         if(messagesSeen < expectedTotalMsgs){
+          // scalastyle:off print-ln
```

Comment:
We can instead remove this `println`, it hasn't really been useful since some initial debugging.

Reply:
+1